apiary.apib

```diff
@@ -399,6 +399,8 @@ We list all backwards-compatible additions here. These are currently available i
 
 ### Latest
 
+- We added `status` on `companies.list`
+- We added `status` on `companies.info`
```

Comment:
Q: Let's maybe create a new header for this and wrap the older changes under their month

Reply:
Probably wise to apply this change in the changelog file and not in the "build" file

Reply:
Oh missed that.